docs/src/app/(private)/experiments/tooltip.tsx

```diff
@@ -188,6 +188,12 @@ export default function TooltipTransitionExperiment() {
       </Tooltip.Root>
 
       <h3>CSS Transition with `@starting-style`</h3>
+      <p>
+        No longer supported due to{' '}
+        <a href="https://github.com/mui/base-ui/pull/992">
+          https://github.com/mui/base-ui/pull/992
+        </a>
+      </p>
```

Comment:
`@starting-style` no longer works with this change. Considering it doesn't handle this issue either, it shouldn't be encouraged. This is better for consistency if they always use `[data-starting-style]` and `[data-ending-style]` hooks anyway.